package.json

```diff
@@ -15,6 +15,7 @@
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
     "astro": "^4.14.3",
+    "data-fns": "^1.1.0",
```

Comment:
```suggestion
```

Seems like an error, we do not use data-fns, but date-fns

Reply:
Ooft, good catch 😅